go-controller/pkg/ovn/pod_selector_address_set.go

```diff
@@ -731,3 +739,31 @@ func deleteAddrSetsWithoutACLRef(predicateIDs *libovsdbops.DbObjectIDs,
 	}
 	return nil
 }
+
+func (bnc *BaseNetworkController) syncAddressSets() error {
```

Comment:
I made it as a receiver function of BaseNetworkController, because it will be needed for all networks downstream.

Reply:
Even if you keep it here now, someone in the future is probably bound to move it to a more appropriate place. How are you going to avoid that? Can't this be moved in your downstream, along with the other differences you must be already handling.

Anyway, we need a more appropriate name for this method because is is also sync'ing ACLs.

Reply:
ok, I will move it to default_network_controller.go and change the name to syncAddressSetsAndAcls()